manual/Remove-contaminated-files-from-tdr.md

```diff
@@ -0,0 +1,168 @@
+# Remove contaminated files from the TDR system
+
+This guides the steps necessary to remove files from the TDR database and S3 buckets.
+
+## SSH tunneling to connect bastion from your local machine - (required to delete from intg/staging database)
+
+1. Install the latest aws cli on your machine.
+2. Create an AWS CLI profile
+   ```
+   aws configure sso
+   aws sso login --profile tdr-integration ( change to the profile you created)
+   ```
+3. Add your ssh key to the bastion
```

Comment:
Can you add a note about how we can alternatively get the public key from our computer and add it to the run bastion github action public key field? 

I think it was something like
cd to the .ssh folder and
cat id_ed25519.pub?